src/test/java/pageobject_model/test/Helpers.java

```diff
@@ -0,0 +1,40 @@
+package pageobject_model.test;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import pageobject_model.page.MainClass;
+
+import java.util.concurrent.TimeUnit;
+
+public class Helpers extends MainClass {
```

Comment:
There too many different logic in helper class, you need to rewrite it and think how you could split or at least simplify this class